app/src/consumer/parser_consumer.c

```diff
@@ -496,16 +509,288 @@ __Z_INLINE parser_error_t parser_getItemRuntime(const parser_context_t *ctx,
         }
         case 5: {
             snprintf(outKey, outKeyLen, "ParaTime");
+            const uint8_t *runId = parser_tx_obj.oasis.runtime.meta.runtime_id;
```

Comment:
nit: you can change the types of runId and chain_context to **const char*** and avoid all the casting from the following lines